conda_build/build.py

```diff
@@ -107,6 +107,10 @@ def have_prefix_files(files):
             # skip symbolic links (as we can on Linux)
             continue
 
+        # dont try to mmap an empty file
+        if os.stat(path).st_size == 0:
```

Comment:
Should this be `lstat`? I'm getting [this build error](https://travis-ci.org/conda-forge/emacs-feedstock/builds/162354897).